src/main/java/com/passpnu/passwordmanager/entity/PasswordEntity.java

```diff
@@ -0,0 +1,65 @@
+package com.passpnu.passwordmanager.entity;
+
+
+import jakarta.persistence.*;
+
+import jakarta.validation.constraints.NotBlank;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.NoArgsConstructor;
+
+@Entity
+@Builder
+@AllArgsConstructor
+@NoArgsConstructor
+@Table(name="password")
+public class PasswordEntity {
+    @Id
+    @GeneratedValue(strategy= GenerationType.AUTO)
+    private Long id;
+
+    @NotBlank
+    @ManyToOne
+    @JoinColumn(name="service_id", nullable = false)
+    private ServiceEntity service;
+
+    @NotBlank
+    private String password;
+
+    @ManyToOne
+    @JoinColumn(name="user_id", nullable = false)
+    private UserEntity user;
+
+    public Long getId() {
```

Comment:
remove boilerplate getters/setters

Reply:
The boilerplate getters/setters are replaced with ```@Getters``` and ```@Setter``` annotations